GSS-Cogs/idpd-frontend-homepage
# Department logo dict

This is to add a way to get different logos for the different publishers. It also loads the publisher name conditionally as not all pubs need the name displayed as well.

There's a default component that will load if the publisher is not recognised.
Also added a change in the font used for publisher to match the designs, 'Helvetica Neue'.

This is just a rough idea and we will most likely change how we get these logos in the future.

**to test**
Head over to the catalogue and see if the logos are working as expected. ONS should just be their logo (which includes their name).
You can remove one of the entries from the dictionary to test if the default component is working.

![image](https://github.com/GSS-Cogs/idpd-frontend-homepage/assets/110108574/45e1a417-6e24-402a-8d00-ff3c9a48643c)
